feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt

```diff
@@ -1,31 +1,72 @@
 package com.epmedu.animeal.favourites
 
+import androidx.activity.compose.BackHandler
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ExperimentalMaterialApi
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Scaffold
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
-import com.epmedu.animeal.favourites.data.model.FavouriteFeedingPoint
 import com.epmedu.animeal.favourites.ui.FavouriteFeedingPointItem
+import com.epmedu.animeal.feeding.data.model.Feeder
+import com.epmedu.animeal.feeding.data.model.FeedingPoint
+import com.epmedu.animeal.feeding.data.model.enum.AnimalPriority
+import com.epmedu.animeal.feeding.data.model.enum.AnimalState
+import com.epmedu.animeal.feeding.presentation.model.FeedingPointModel
+import com.epmedu.animeal.feeding.presentation.model.MapLocation
+import com.epmedu.animeal.feeding.presentation.model.toFeedStatus
+import com.epmedu.animeal.feeding.presentation.ui.FeedingPointSheetContent
+import com.epmedu.animeal.foundation.dialog.bottomsheet.*
 import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.switch.AnimalType
 import com.epmedu.animeal.foundation.theme.AnimealTheme
 import com.epmedu.animeal.foundation.theme.bottomBarPadding
 import com.epmedu.animeal.foundation.topbar.TopBar
 import com.epmedu.animeal.resources.R
+import kotlinx.coroutines.launch
 
 @Composable
 internal fun FavouritesScreenUI(
     state: FavouritesState,
+    bottomSheetState: HomeBottomSheetState,
     onEvent: (FavouritesScreenEvent) -> Unit
 ) {
+
+    val (contentAlpha: Float, buttonAlpha: Float) = bottomSheetState.contentAlphaButtonAlpha()
+
+    val scope = rememberCoroutineScope()
+
+    BackHandler(enabled = bottomSheetState.isVisible) {
+        scope.launch { bottomSheetState.hide() }
+    }
+
+    ScreenScaffold(
+        bottomSheetState,
+        state,
+        contentAlpha,
+        onEvent,
+        buttonAlpha
+    )
+}
+
+@Composable
+private fun ScreenScaffold(
+    bottomSheetState: HomeBottomSheetState,
```

Comment:
Hi! The bottom sheet looks sharp, the only issue I see is that is displaying some bottom padding making it different from mapBottomSheet so please remove the '.bottomBarPadding()'. And maybe put contentAlpha and buttonAlpha in order?
Nice job!